### DISCUSSION TITLE
Phpunit.xml server variable not being read when ray config not published

### DISCUSSION THREAD

**Discussion by bsterckx** [spatie/laravel-ray#104] [Q&A] [answered]
Hi,

The product is working well. I just have one thing that is unclear. 
If I **don't** publish the ray config via:
`php artisan ray:publish-config`

The server variables in phpunit.xml seem to be ignored:
      `<server name="RAY_ENABLED" value="false"/>`
     `<server name="SEND_LOG_CALLS_TO_RAY" value="false"/>`

Just wondering if this is expected behaviour?

(By ignoring these variables and not having the ray app running, my unit tests which log to a file get delayed because ray can't connect to the app).

After publishing the config file everything works fine, I just find it a bit strange I need to publish a generic config file which I don't even try to change. 

Thanks for another wonderfull Spatie product!

Kind regards,

Benny

**Comment by freekmurze** [comment, marked as answer]
This is expected. env variables will only be used if there is a config file.
